### DISCUSSION TITLE
Beatmap availability is not checked in multiplayer

### DISCUSSION THREAD

**Discussion by Sinamer** [ppy/osu#15426] [Q&A] [answered]
"No map" icon is not shown for players in the lobby, it is also possible to click "Ready" button even if the map is not downloaded:
![osu_2021-11-02_19-51-57](https://user-images.githubusercontent.com/53346335/139919486-0027aae8-0f46-4717-ae03-53e3bbbb65f8.png)

**Comment by peppy** [comment, marked as answer]
Not able to reproduce this. Your room may have just been in a bad state?

**Reply by peppy** [reply]
May be fixed via #15432.

**Reply by Sinamer** [reply]
Can confirm it is no longer an issue (running osu! with that PR applied):
![osu_2021-11-02_21-44-27](https://user-images.githubusercontent.com/53346335/139942641-5de269c4-076e-4889-80d5-244599893c59.png)

**Reply by peppy** [reply]
Would ask that until we have proper checks in place that you please do not connect to multiplayer servers using self-compiled releases.
